app/domain/logs.rb

```diff
@@ -290,6 +290,26 @@ module AuthnJwt
         code: "CONJ00056D"
       )
 
+      FetchingJwtMandatoryClaimsToValidate = ::Util::TrackableLogMessageClass.new(
+        msg: "Fetching JWT mandatory claims to validate",
```

Comment:
do we want to say from where?

Reply:
We actually decide here which mandatory claims to validate base on hardcoded logic + the token content (in the future the logic may be configured in the policy)

Do you think we should change this message ?

Reply:
i think it will enhance our supportability if the log would say where the claims are taken from, especially if it can be from several places.

Reply:
Since i am printing later which claims are mandatory to validate, i don want to print here what is the logic of how i decide that. 

The message will look something like this:

```
Fetching JWT mandatory claims to validate, according to the following logic:
If `iss, exp, nbf, iat` are exist in the token they are mandatory
If not, we ignore them
```

@hilagross @sashaCher @orenbm Ignore the english, do we want to print this king of message ?

Reply:
I'm not sure we should print such massage, 
first what is mandatory claims? there is no mandatory claim in JWT RFC, each application can decide what to send in its JWT token.
IMO we should print only "Fetching JWT claims to validate" and drop the mandatory.
I think this should be given as info in the documentation.

Reply:
no this message is not king at all. we shouldn't log the logic. however, we can log the `Fetching JWT mandatory claims to validate` message after we perform the logic and then say something like `Fetching JWT mandatory claims to validate for the token` (if i understand correctly)

Reply:
Oh I like @hilagross 's approach! 
Let's drop the word `mandatory` from all the classes, it makes much more sense now. Class `FetchJwtClaimsToValidate` that goes over the optional claims and decides which ones should be validated is much clearer than `FetchJwtMandatoryClaimsToValidate`

Reply:
Agree with @hilagross .